docs/README.md

```diff
@@ -0,0 +1,47 @@
+# Testing github pages locally
+
+This includes steps to test local changes related to github pages.
+
+Note: Current repo github pages leverages  `jekyll-theme-minimal` theme.
+
+* Install dependencies
+
+```bash
+sudo apt update
+sudo apt install ruby-full build-essential zlib1g-dev
+```
+
+* Create Gemfile under `docs` directory
+
+```bash
+cat <<EOF > Gemfile
+source "https://rubygems.org"
+
+gem "jekyll", "~> 4.3.0"
+gem "webrick"
+gem "jekyll-theme-minimal"
+EOF
+```
+
+* Install bundle & dependencies
+
+```bash
+sudo gem install jekyll bundler
+cd docs
+bundle install
+```
+
+* Update `_config.yml`
```

Comment:
The Jekyll command line allows you to specify multiple config files, with later config files overriding values of earlier config files. Could this be used to avoid needing to temporarily modify the `_config.yml` file?